wherehows-kafka/src/main/java/wherehows/actors/KafkaWorker.java

```diff
@@ -30,28 +32,35 @@
 
   public static boolean RUNNING = true;
 
+  public static final String WORKER_START = "WORKER_START";
+
+  private final int _consumer_poll_interval = 1000;
+
   private final String _topic;
 
+  private TopicPartition _partition;  // assuming there is only one partition assigned to each worker
+
   private final KafkaConsumer<String, IndexedRecord> _consumer;
 
   private final KafkaMessageProcessor _processor;
 
-  private final int _consumer_poll_interval = 1000;
+  private boolean _alive;  // status of this worker
```

Comment:
Why do we need this?

Reply:
It's not used for now. I though maybe the master would want to check the status of each worker and restart a certain consumer if necessary. I will take this our. We may add this status check or heartbeat later on.